src/overlays/actors/ovl_Boss_Tw/z_boss_tw.h

```diff
@@ -6,9 +6,138 @@
 
 struct BossTw;
 
+typedef void (*BossTW_ActionFunc)(struct BossTw* this, GlobalContext* globalCtx);
+
+#define CS_TIMER_1 0
```

Comment:
These should probably be an enum with the last entry being the number of work variables.  BossFD has this already if you want to reference something.  This also applies to fWork.